scripts/download-smithy-models.sh

```diff
@@ -1,5 +1,6 @@
-git clone git@github.com:aws/aws-sdk-js-v3.git --depth=1
+git clone https://github.com/aws/aws-sdk-js-v3.git --depth=1
```

Comment:
Why https protocol? I don't have strong opinions but was this done for a specific reason?

Reply:
I'm not sure why but the git protocol didn't work for me. I will revert that too since I don't really have to use it.